index.js

```diff
@@ -128,6 +130,16 @@ async function tagVersion(args) {
   return 'Done!';
 }
 
+// TODO:
+// - list all branches on upstream
+// - sort by RELEASE_PREFIX
+// - select one previous to current version
+// - git shortlog currentBranch...previousBranch
+// - generate github url to this effect
+async function createReleaseNotes(args) {
```

Comment:
I'm not sure if this should create the release notes (output markdown for example) but I'd definitely be ok with it showing the log between current tag and previous tag (which may be a simpler way to get at this info). Depending on branches for this info would be error prone but diff `<previous_tag>...<current_tag> would work`

Reply:
cool, yeah perhaps "notes" is misleading. just some sugar to output the ```git shortlog``` between the latest releases.

i'll see if i can't find another way to get the previous tag; semver doesn't have a ```decr``` (which is probably good)

Reply:
Yeah at least we would know what the previous version is when you are running the tool. For example: you are doing either a tag or a cut with major/minor flag so we know how to get back to where you were... currently v1.2.3 doing a minor cut, etc